### PR TITLE
Fix parsing modifiers with context of block/elem

### PR DESCRIPTION
**Pull request by Yeti-or**
Previous decision was wrong we actually need to extract main entity
Check this issue: https://github.com/bem/webpack-bem-loader/issues/64
Natural dependencies were broken

### CODE REVIEW

File: packages/import-notation/index.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ function parse(importString, ctx) {
             if(!main.block) {
                 main.block = ctx.block;
                 main.elem || ctx.elem && (main.elem = ctx.elem);
+                acc.add(main);
```
Comment (Vittly) [inline packages/import-notation/index.js]: need to fix jsdoc "[ctx] - entity to restore `block` part" > "[ctx] - entity to restore `block`/`elem` parts"

File: packages/import-notation/README.md [hunk 2]
```diff
@@ -63,14 +63,11 @@ Context allows to extract portion of entities.
 ```js
 var enties = parse('m:theme=normal', { block: 'button' });
 
-// → [ { block: 'button', mod: { name: 'theme' } },
+// → [ { block: 'button' },
```
Comment (Vittly) [inline packages/import-notation/README.md:66]: can we enclose `ctx` in `[]` in table above like `[ctx]` ? To mark it optional

**Comment by Vittly** [comment]
🆗

File: packages/import-notation/index.js (hunk 1 above)
Comment (qfox) [reply packages/import-notation/index.js]: We call it scope in the rest modules. Probably better to rename it here too.
block/elem is the base entity name (not part). But can be used in "part of import string" meaning.

**Review by qfox — APPROVED** [review]

**Pull request merged**